controllers/homeRoutes.js

```diff
@@ -2,6 +2,51 @@
 const router = require('express').Router();
 const withAuthorization = require('../utils/auth');
 
+
```

Comment:
moved routes to the top and array to the bottom; so the array isn't in our face when trying to look at routes.